packages/liferay-npm-scripts/__tests__/scripts/theme.js

```diff
@@ -8,10 +8,12 @@ const os = require('os');
 const path = require('path');
 
 const FIXTURES = path.resolve(__dirname, '../../__fixtures__/scripts/theme');
-const APPS = path.join(FIXTURES, 'modules/apps');
+const MODULES = path.join(FIXTURES, 'modules');
```

Comment:
Note that this commit is not the smallest possible edit that I could have made, but I figured it makes sense to treat "modules" as the root instead of "modules/apps" now that we're dealing with something at that level (ie. "modules/node_modules").